cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -188,6 +172,13 @@ def _configure_ssl(self):
                 common.chmod('444', POSTGRESQL_CA_CERT_PATH)
 
     def _configure(self):
+        if config[POSTGRESQL_CLIENT]['azure_domain']:
+            # Need to save the proper username for cloudify-rest.conf
+            # e.g for Azure implementation - test@test_domain
+            config[POSTGRESQL_CLIENT]['username'] += \
```

Comment:
Shouldn't they just be putting the username in as that, verbatim- rather than us adding yet another new config key for every supported backend provider?

Reply:
I thought of that, but since they would need to add it in 2 locations I thought it would be better to have it this way.
I'm not married to the idea of course

Reply:
I think they should just be putting it in verbatim- otherwise, we end up with azure_domain, then aws_rds_domain, then gcp_something_domain, etc.

Reply:
tbh I only noticed this in Azure (Probably Active Directory stuff).
But sure, I'll update.